db/buildData.js

```diff
@@ -0,0 +1,100 @@
+const path = require('path');
+const faker = require('faker');
+const buzzphrase = require('buzzphrase');
+const createCsvWriter = require('csv-writer').createArrayCsvWriter;
+const shortid = require('shortid');
+
+const CHUNK_SIZE = 100000;
+const MAX_TRAITS = 100;
+
+const setupTraits = async () => {
+  const csvWriter = createCsvWriter({
+    path: path.resolve(__dirname, 'data', 'traits.csv'),
+    header: ['_key', 'name']
+  });
+  let traits = [];
+
+  while (traits.length < MAX_TRAITS) {
+    let trait = buzzphrase.get({ format: '{a}' });
+    if (!traits.includes(trait)) {
+      traits.push([shortid.generate(), trait]);
+    }
+  }
+
+  await csvWriter.writeRecords(traits);
+
+  return traits;
+};
+
+const setupGames = async (n) => {
+  let chunks = Math.floor(n / CHUNK_SIZE) - 1;
+  let count = 0;
+  let games = [];
+  try {
+    for (let i = 0; i <= chunks; i++) {
+      const csvWriter = createCsvWriter({
+        path: path.resolve(__dirname, 'data', `games${i}.csv`),
+        header: ['_key', 'product']
```

Comment:
I made & deleted a comment earlier about lack of product_ids, but now I see that it's here, named as product instead. Maybe a comment above each function summarizing its general function (since the name `setupGames` could be a bit vague) would help clarify some functionality without too much digging around :)